test/Microsoft.ML.Tests/Transformers/ConcatTests.cs

```diff
@@ -17,6 +18,25 @@ public ConcatTests(ITestOutputHelper output) : base(output)
         {
         }
 
+        [Fact]
+        void TestConcatNoInputColumns()
+        {
+            var thrown = false;
+
+            try
+            {
+                var pipe = ML.Transforms.Concatenate("Features");
```

Comment:
>Concatenate [](start = 41, length = 11)

maybe it make sense to remove `params` from inputColumnNames in Concatenate method?